ports/servoshell/desktop/app.rs

```diff
@@ -114,6 +113,7 @@ impl App {
         self.suspended.set(false);
         let (_, window) = self.windows.iter().next().unwrap();
 
+        #[cfg(feature = "webxr")]
```

Comment:
I'm unclear why we want a webxr feature for servoshell at all. It's just our sample embedding; I think we should minimize the customizations that are possible for it.